Description:Fixed ruff issue
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 python3-huggingface-hub (0.29.3-1) UNRELEASED; urgency=medium
 .
   * Initial release. (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: anudeep <anudeep@unknown>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2025-03-24

--- python3-huggingface-hub-0.29.3.orig/pyproject.toml
+++ python3-huggingface-hub-0.29.3/pyproject.toml
@@ -40,10 +40,10 @@ exclude = [
 line-length = 119
 # Ignored rules:
 # "E501" -> line length violation
-lint.ignore = ["E501"]
-lint.select = ["E", "F", "I", "W"]
+ignore = ["E501"]
+select = ["E", "F", "I", "W"]
 
-[tool.ruff.lint.isort]
+[tool.ruff.isort]
 known-first-party = ["huggingface_hub"]
 lines-after-imports = 2
 
